apps/storefront/src/pages/QuickOrder/utils.ts

```diff
@@ -157,3 +158,18 @@ export const addCartProductToVerify = async (
     return handleVerifyProduct(product, b3lang).isVerify;
   });
 };
+
+export const parseOptionList = (options: string[] | undefined): ShoppingListAddProductOption[] => {
```

Comment:
💅 Oh, there was no need to move it to `utils`, specially if it is not used elsewhere. Just outside the component, but in the same file.